### PR TITLE
v0.1.4: Mitigate the invisible rectangles issue

### PR DESCRIPTION
**Pull request by HollowMan6**
Related PR: https://github.com/ThibaultJanBeyer/DragSelect/pull/152
Issue: https://github.com/mit-cml/workspace-multiselect/pull/6#issuecomment-1315829670

Now the issue is mitigated, the click-selection should be OK.
GIF 2022-11-22 1-49-14

Signed-off-by: Hollow Man <hollowman@opensuse.org>

### CODE REVIEW

**Comment by HollowMan6** [comment]
Progress: Wait for https://github.com/ThibaultJanBeyer/DragSelect/pull/165 to get merged

**Comment by HollowMan6** [comment]
Hi @ewpatton / @mark-friedman ! A new release for DragSelect is now ready: https://github.com/ThibaultJanBeyer/DragSelect/releases/tag/v2.6.0

Would you please help merge this PR and make the new release as well?

File: package.json [hunk 1]
```diff
@@ -39,12 +39,12 @@
...
   "dependencies": {
-    "dragselect": "^2.5.4"
+    "dragselect": "^2.6.0"
   },
   "devDependencies": {
     "@blockly/dev-scripts": "^1.2.28",
-    "@blockly/dev-tools": "^5.0.2",
-    "blockly": "^9.0.0"
+    "@blockly/dev-tools": "^5.0.5",
+    "blockly": "^9.2.1"
```
Comment (ewpatton) [inline package.json:47]: Is bumping the Blockly version necessary here? I feel like the change to dragselect should be sufficient and there's no reason to disallow early Blockly versions if the code is compatible.

Comment (HollowMan6) [reply package.json:47]: Hi! I admit that bumping the Blockly version is not something necessary, here I just want to make sure that the latest Block version works with this plugin, and it's only for development purpose. (`devDependencies` specifies packages that are only needed for local development and testing https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file ). I'm not bumping the blockly version in peerDependencies, so the change here is not related to disallow early Blockly versions.

Comment (HollowMan6) [reply package.json:47]: You can find the table here more helpful: https://www.geeksforgeeks.org/difference-between-dependencies-devdependencies-and-peerdependencies/

**Review by ewpatton — APPROVED** [review]

**Pull request merged**